internal/api/web/views/components/quiz_components/question_content.templ

```diff
@@ -0,0 +1,15 @@
+package quiz_components
+
+import "github.com/Molnes/Nyhetsjeger/internal/data/questions"
+
+templ QuizContent (question questions.Question) {
+	<section id="outer-image" class="flex content-center justify-center mt-4">
```

Comment:
not sure about the correct use of sections here  
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section  
shouldnt the img and question be in one?